server/enterprise/message_export/csv_export/csv_export.go

```diff
@@ -404,27 +401,13 @@ func postToRow(post *model.MessageExport, createTime *int64, message string) []s
 }
 
 func attachmentToRow(post *model.MessageExport, attachment *model.FileInfo) []string {
-	row := postToRow(post, post.PostCreateAt, *post.PostMessage)
-
-	attachmentEntry := fmt.Sprintf("%s (files/%s/%s-%s)", attachment.Name, *post.PostId, attachment.Id, path.Base(attachment.Path))
-	attachmentMessage := "attachment"
-	userType := row[len(row)-2]
-
-	if attachment.DeleteAt > 0 && post.PostDeleteAt != nil {
-		deleteRow := postToRow(post, post.PostDeleteAt, *post.PostMessage)
-		row = append(
-			deleteRow[:len(deleteRow)-4],
-			attachmentEntry,
-			"deleted "+attachmentMessage,
-			userType,
-		)
-	} else {
-		row = append(
-			row[:len(row)-4],
-			attachmentEntry,
-			attachmentMessage,
-			userType,
-		)
+	message := strings.TrimSpace(fmt.Sprintf("%s (files/%s/%s-%s)", attachment.Name, *post.PostId, attachment.Id, path.Base(attachment.Path)))
+	createAt := post.PostCreateAt
+	postType := "attachment"
+	if common_export.IsDeletedMsg(post) {
+		createAt = model.NewPointer(attachment.DeleteAt)
+		postType = "deleted attachment"
```

Comment:
You mean instead of constants? No reason, it was always like this. I think its only the csv that has this column. I can turn it into a constant in the csv-rework PR I'm doing now. 👍